keyboards/walletburner/cajal/keymaps/via_ortho/keymap.c

```diff
@@ -0,0 +1,92 @@
+/* Copyright 2020 Worldspawn <mcmancuso@gmail.com>
```

Comment:
This keymap should be removed. 

Generally, you shouldn't need or have multiple via keymaps, as via should be able to handle different layouts in it's UI

Reply:
Good point - thank you.